### DISCUSSION TITLE
240403

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#112] [팀회의]
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
- teamId, 팀 정보, 유저정보를 프롭으로 받을건지 아니면 모달 내부에서 불러서 사용할건지 의논 필요 
## 어제 한 일
- x
## 오늘 할 일
- 베포, 위에 논의사항 정해지면 논의사항해결하고 그룹을 해결해야함
## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항

## 어제 한 일
- drag & drop 완성
- 오류 수정

## 오늘 할 일
- 게시글 pin 기능 
- 상세보기 페이지 (?)
- 팀 새로운 api 반영
- 배포하기

## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
빌드 시 오류 수정(그렇지만 합치면 또 일어나지 않을까 싶습니다!) 
네브바 팀 초대 알람 모달 완성
모달에 들어가는 시간 설정 부분 UI 
## 오늘 할 일
모달에 캘린더 시간 부분 값 연결시키기 , 빌드시 발생하는 오류 수정하기 
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
members 완성
## 오늘 할 일
히스토리 관리 모달 api 연결, 변경사항으로 인한 signup 에러 해결, 빌드 시 에러 확인 후 해결
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
